supertokens_python/recipe/emailverification/interfaces.py

```diff
@@ -121,8 +125,7 @@ def to_json(self) -> Dict[str, Any]:
 
 
 class EmailVerifyPostInvalidTokenError(APIResponse):
-    def __init__(self):
-        self.status = "EMAIL_VERIFICATION_INVALID_TOKEN_ERROR"
+    status = "EMAIL_VERIFICATION_INVALID_TOKEN_ERROR"
```

Comment:
why this change?

Reply:
Hmm. it's not required. 

Restored it.